distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc

```diff
@@ -1379,6 +1379,483 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+                       <ns2:pos>27 32 </ns2:pos>
+                   </gml:LineString>
+               </ogc:Contains>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+</csw:GetRecords>
+----
+===== `XPath Operation Example`
+A xpath query for records with a xpath property with a value like "TextToMatch".
+[source,xml,linenums]
+----
+ <csw:GetRecords xmlns:ows="http://www.opengis.net/ows" xmlns:ns2="http://www.w3.org/1999/xlink"
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:PropertyIsLike wildCard="*" singleChar="#" escapeChar="!">
+                   <ogc:PropertyName>/Record/title</ogc:PropertyName>
+                   <ogc:Literal>TextToMatch</ogc:Literal>
+               </ogc:PropertyIsLike>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+</csw:GetRecords>
+----
+===== `Logical Operation Example`
+A query for records which were created before a specific date AND contain a string like "PlainXml"
+[source,xml,linenums]
+----
+<csw:GetRecords xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+         xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+         xmlns:ns4="http://www.w3.org/1999/xlink" xmlns:ns5="http://www.w3.org/2001/SMIL20/"
+         xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:ows="http://www.opengis.net/ows"
+         xmlns:dct="http://purl.org/dc/terms/" xmlns:ns9="http://www.w3.org/2001/SMIL20/Language"
+         resultType="results"  outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+         startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+ <csw:Query typeNames="csw:Record">
+   <csw:ElementSetName>full</csw:ElementSetName>
+   <csw:Constraint version="1.1.0">
+     <ogc:Filter>
+       <ogc:And>
+         <ogc:PropertyIsLessThan>
+           <ogc:PropertyName>created</ogc:PropertyName>
+           <ogc:Literal>2012-12-26T10:10:19.000-07:00</ogc:Literal>
+         </ogc:PropertyIsLessThan>
+         <ogc:PropertyIsLike wildCard="*" singleChar="#" escapeChar="!">
+           <ogc:PropertyName>AnyText</ogc:PropertyName>
+           <ogc:Literal>PlainXml</ogc:Literal>
+         </ogc:PropertyIsLike>
+       </ogc:And>
+     </ogc:Filter>
+   </csw:Constraint>
+   <ogc:SortBy>
```

Comment:
Probably should add to the description regarding this sorting or remove it from the example query.